examples/get-content-from-multiple-elements/index.js

```diff
@@ -0,0 +1,21 @@
+import { Selector } from 'testcafe';
+
+fixture `Get content from multiple elements`
+    .page `./index.html`;
+
+async function getTextFromElements (selector) {
+    const result = [];
+
+    for (let i = 0; i < await selector.count; i++) {
+        result.push(await selector.nth(i).textContent);
+    }
+
+    return result;
```

Comment:
At present, the [Smart Assertion Query Mechanism](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/assert.html#smart-assertion-query-mechanism) is not used.

If you rewrite the code using ClientFunction, then it will be used.

Reply:
I tried to make use of the Smart Assertion Query Mechanism using Selectors. Would this solution be suitable?